packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -273,9 +277,9 @@ describe("Odsp Error", () => {
 				error.message.includes("TestMessage"),
 				"message should contain original message",
 			);
-			assert.equal(error.canRetry, false, "canRetry should be false");
+			assert.equal((error as AuthorizationError).canRetry, false, "canRetry should be false");
```

Comment:
Tangent, not for this PR: seems like we need helper function to do type narrowing on all these error types.

Reply:
Yeah. That was my biggest "next steps" takeaway from this effort. We need better patterns for our error handling and propagation.